lib/testing/src/lib/process-services-cloud/actions/process.services.ts

```diff
@@ -58,6 +58,17 @@ export class ProcessServices {
         return processInstance;
     }
 
+    async createProcessInstanceWithVariables(processDefName, appName, variables: any, processInstanceName?: string) {
+        const processDefinition = await this.processDefinitionsService.getProcessDefinitionByName(processDefName, appName);
+        const processInstance = await this.processInstancesService.createProcessInstance(processDefinition.entry.key, appName, {
+            name: processInstanceName ? processInstanceName : StringUtil.generateRandomString(),
+            businessKey: StringUtil.generateRandomString(),
```

Comment:
why are we passing a random businessKey ?

Reply:
I don't know, i kept it the same as for createProcessInstance method.